lib/evaluation/evaluationClientWrapper.ts

```diff
@@ -797,7 +874,7 @@ export class EvaluationClientWrapper {
       presentationSubmissionLocation?: PresentationSubmissionLocation;
     },
   ): PresentationSubmission {
-    if (!this._client.results) {
+    if (!this._client.results || this._client.results.length === 0) {
```

Comment:
I am not confident this is a correct change. Did you check it? Are we sure that evaluate() always returns arrays in all cases

Reply:
Will update the client to start  this._results as undefined

Reply:
Yes that is why I asked to look into whether that always happens

Reply:
Unless an empty WrappedVerifiableCredential is passed in. Added a different error message for that
>The WrappedVerifiableCredentials input array is empty